static/src/javascripts/projects/commercial/modules/messenger/background.js

```diff
@@ -135,6 +158,12 @@ const setBackground = (specs: AdSpec, adSlot: HTMLElement): Promise<any> => {
             specs.scrollType
         }`;
 
+        // This is improted from _creatives.css
+        if (specs.scrollType === 'fixed') background.style.position = 'fixed';
+
+        if (specs.scrollType === 'parallax')
+            background.style.position = 'absolute';
+
```

Comment:
I remember this being an important file for page skins. They're not technically native ads in the same way as some others but will use Ad Manager. Has this been checked against PageSkins and TruSkins?

Reply:
This has not been checked with PageSkins or TruSkins, yet. I'll get in touch with relevant parties.

Reply:
You can actually do it yourself locally using an ad test. There should be a couple of them that I'll send over

Reply:
That'll go in a further PR